vms/avm/index_test.go

```diff
@@ -258,7 +258,7 @@ func buildUTXO(utxoID avax.UTXOID, txAssetID avax.Asset, addr ids.ShortID) *avax
 	}
 }
 
-func buildTX(utxoID avax.UTXOID, txAssetID avax.Asset, address ...ids.ShortID) *txs.Tx {
+func buildTX(chainID ids.ID, utxoID avax.UTXOID, txAssetID avax.Asset, address ...ids.ShortID) *txs.Tx {
```

Comment:
consider passing env here

Reply:
I think that's a bit overkill tbh- we'll probably move this to a helper in a future PR